lib/stella/helpers.rb

```diff
@@ -0,0 +1,103 @@
+require 'typhoeus/adapters/faraday'
+
+module Stella
+  module Helpers
+    extend ActiveSupport::Concern
+
+    @@types = []
+
+    included do
+      if defined? Rails
```

Comment:
I'm not sure how best to deal with this Rails dependency. Environment versioning on index names seems quite important for the lib to include but I guess it's not a hard requirement - the host repo could override the index naming scheme with that perhaps.